shared/config.gradle

```diff
@@ -21,6 +21,11 @@ nativeUtils.wpi.addWarningsAsErrors()
 
 nativeUtils.setSinglePrintPerPlatform()
 
+nativeUtils.platformConfigs.named("osxx64-64").configure {
+    it.cppCompiler.args << "-mmacosx-version-min=10.9"
```

Comment:
lol, oops -- but I think it'll just select the minimum that it can. Once the build finishes if it works I can change it to 10.13.